pom.xml

```diff
@@ -56,6 +56,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>jdk-tool</artifactId>
+            <version>1.3</version>
```

Comment:
Is it really needed?

Reply:
in order to validate the java 11 support, the plugin is built against `2.164.1` of Jenkins Core. But the `JDKTool*` classes were removed from the core in `2.111`. That is why we had errors in the first commits. 
It's not great, I agree but yes it is really needed.

Reply:
Uh oh, created https://issues.jenkins-ci.org/browse/JENKINS-58672

Reply:
Not a blocker for this PR for the record